python/xt/dune/xt/test/hypothesis_strategies.py

```
@@ -0,0 +1,260 @@
+            ]
+        )
+    )
+    extents = draw(
+        st.tuples(
+            *[
+                st.floats(min_extent, max_extent, allow_nan=False, allow_infinity=False)
+                for _ in range(dim)
+            ]
+        )
+    )
+    upper = tuple(ll + e for ll, e in zip(lower, extents))
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)